tests/test_reporting.py

```diff
@@ -62,7 +62,7 @@ def test_reporter(test_mp):
 
     # …and expected values
     vom = rep.get(rep.full_key('ACT')) * rep.get(rep.full_key('var_cost'))
-    assert_xr_equal(vom, rep.get(vom_key))
+    assert_series_equal(vom, rep.get(vom_key))
```

Comment:
I think we should add `ixmp.testing.assert_qty_equal(a, b)` that tests and maybe casts a and b to the current value of `ixmp.reporting.utils.Quantity`. That would also avoid this:
https://github.com/iiasa/ixmp/blob/d74c49326cc98021037af0122a484e4bbcd84114/tests/test_reporting.py#L444-L448